ui/src/App.js

```diff
@@ -354,6 +357,28 @@ class App extends Component {
     return newValueList;
   }
 
+  removeFacet(facetValue) {
+    this.state.facets.delete(facetValue);
```

Comment:
Array.filter works because it returns the array. Unfortunately Map.delete returns a boolean. I suppose if you really want to treat this.state as immutable, we'd create a local variable and set it to a clone of the map.

Reply:
Oops ok, whatever you decide is fine